### PR TITLE
Feature/add data set

### PR DESCRIPTION
**Pull request by dlouise64**
Charts for data set
Screen Shot 2019-04-09 at 1 58 06 PM

### CODE REVIEW

**Comment by dlouise64** [comment]
@verbman The first run through of this, can you please test this and let me know if there's anything missing 👍

**Comment by Br3nda** [comment]
I'm getting this
```
./src/components/Chart.js
Module not found: Can't resolve 'react-plotly.js' in '/home/brenda/projects/Porridge/src/components'
```

**Comment by Br3nda** [comment]
which nodejs version should i be using?

Also there seems to be both a yarn.lock and a package-lock.json
the readme sayings to use npm.

**Comment by Br3nda** [comment]
if you put a file called `.nvmrc` in the folder, it'll tell nvm which to use for this repo

File: package.json [hunk 1]
```diff
@@ -26,5 +26,8 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
```
Comment (mischa-s) [inline package.json:31]: :sparkles:

File: src/App.js [hunk 2]
```diff
@@ -5,22 +5,45 @@ import Simulation from './containers/Simulation';
...
-        <AccordionItem
-          key={example.firstName}
-          person={example}
-        />
-      )}
-      )}
-      </div>
-    </Paper>
-  );
+import AppBar from '@material-ui/core/AppBar';
```
Comment (mischa-s) [inline src/App.js:8]: :scroll:

File: src/charts/Data.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+var DataSet = [
```
Comment (mischa-s) [inline src/charts/Data.js]: const?

File: src/openfisca/Data.js [hunk 4]
```diff
@@ -195,259 +195,7 @@ const DATA = [
       ['2023-09-06', null],
       ['2024-09-06', null]
     ]
-  },
```
Comment (mischa-s) [inline src/openfisca/Data.js:198]: Is everything below here duplicate data? why is it being deleted?

File: src/utilities/groupAndCount.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    result = [];
+
+  while (i--) {
+    if (!array.includes(v)) result.push(v);
+    ++v;
+  }
+  return result;
+}
+
+function sortObj(data, group, missing_numbers) {
```
Comment (mischa-s) [inline src/utilities/groupAndCount.js:15]: :surfing_woman:  Lodashing it up :running_shirt_with_sash:

**Comment by dlouise64** [comment]
@Br3nda @JacOng17 forgot to tell you run `npm install react-plotly.js plotly.js`

File: src/containers/Simulation.js [hunk 6]
```diff
@@ -1,39 +1,26 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState } from 'react'
 
 import Card from '@material-ui/core/Card';
 import Chart from '../components/Chart';
+import DataSet from '../charts/Data';
+import _ from 'lodash';
```
Comment (mischa-s) [inline src/containers/Simulation.js:7]: :sparkles:

File: src/components/Chart.js [hunk 7]
```diff
@@ -1,31 +1,68 @@
...
-        type: 'bar'
-      };
-
-      var trace2 = {
-        x: ageRange,
-        y: [12, 18, 29],
-        name: 'Proposed State',
-        type: 'bar'
-      };
+  function getGroup(data, filter, missing_numbers) {
```
Comment (mischa-s) [inline src/components/Chart.js:7]: :sparkler:

**Review by mischa-s — CHANGES_REQUESTED** [review]
Great work :sparkles: ! Two questions are `var` vs `const` and deleting demo data

File: src/charts/Data.js [hunk 8]
```diff
@@ -1,4 +1,4 @@
-var DataSet = [
+const DataSet = [
```
Comment (mischa-s) [inline src/charts/Data.js:1]: :sparkles:

**Review by JacOng17 — COMMENTED** [review]
Any reason why it is compiled with warnings?

./src/utilities/groupAndCount.js

  Line 45:  Expected to return a value in arrow function  array-callback-return

./src/openfisca/Query.js

  Line 8:   Expected to return a value in arrow function  array-callback-return

  Line 10:  Expected to return a value in arrow function  array-callback-return

**Comment by dlouise64** [comment]
@JacOng17 thanks will fix :)

**Comment by mischa-s** [comment]
Any reason `react-plotly.js` `plotly.js` haven't been added to the package.json?

File: src/openfisca/Data.js (hunk 4 above)
Comment (dlouise64) [reply src/openfisca/Data.js:198]: @mischa-s It was made up data. I removed it to keep the tests clean.

Comment (mischa-s) [reply src/openfisca/Data.js:198]: :+1:

**Review by mischa-s — APPROVED** [review]
schweeettt :candy: :sparkles:

**Review by Br3nda — APPROVED** [review]

**Pull request merged**